### PR TITLE
streamingccl/logical: add vtable info for writer

### PR DESCRIPTION
**Pull request by dt**
Release note: none.
Epic: none.

### CODE REVIEW

**Comment by dt** [comment]
first pass at some of the numbers we discussed. I don't think we count errors yet since we don't retry them yet?

Still need to update the zip tests to include the columns.

File: pkg/sql/crdb_internal.go [hunk 1]
```diff
@@ -9205,7 +9205,19 @@ var crdbInternalLDRProcessorTable = virtualSchemaTable{
...
+	last_recv_wait INTERVAL,
+	flushes INT,
+	flush_time INTERVAL,
+	last_flush_time INTERVAL,
+	last_flush_kvs INT,
+	last_flush_bytes INT,
+	last_flush_slowest_batch INTERVAL,
+	cur_flush_running INTERVAL,
+	cur_flush_kvs INT,
+	cur_flush_processed INT,
```
Comment (msbutler) [inline pkg/sql/crdb_internal.go:9219]: nit: i think this reduces the width of table and makes the col names slightly clearer:
s/cur_flush_kvs/cur_flush_total_kvs/r
s/cur_flush_processed/cur_flush_kvs/r

then it's clearer that that cur_flush_kvs and cur_flush_batches are running sums.

**Review by msbutler — COMMENTED** [review]
Nice! mostly nits, one blocking comment about Flushes.Nanos not getting updated.

**Comment by dt** [comment]
RFAL

File: pkg/repstream/streampb/empty.go [hunk 2]
```diff
@@ -121,4 +122,84 @@ type DebugLogicalConsumerStatus struct {
...
+	d.mu.stats.Flushes.Bytes += byteSize
+	d.mu.stats.Flushes.Batches += d.mu.stats.Flushes.Current.Batches
+
+	d.mu.stats.Flushes.Last.Nanos = totalNanos
+	d.mu.stats.Flushes.Last.Batches = d.mu.stats.Flushes.Current.Batches
+	d.mu.stats.Flushes.Last.SlowestBatchNanos = d.mu.stats.Flushes.Current.SlowestBatchNanos
+	d.mu.stats.Flushes.Last.KVs = keyCount
+	d.mu.stats.Flushes.Last.Bytes = byteSize
+
+	d.mu.stats.Flushes.Current.StartedUnixMicros = 0
```
Comment (msbutler) [inline pkg/repstream/streampb/empty.go:199]: looks like you need to reset `d.mu.stats.Flushes.Current.SlowestBatchNanos`

File: pkg/sql/crdb_internal.go [hunk 3]
```diff
@@ -9216,10 +9232,44 @@ CREATE TABLE crdb_internal.logical_replication_node_processors (
...
+				tree.NewDInt(tree.DInt(status.Flushes.Batches)),
+				dur(status.Flushes.Last.Nanos),
+				tree.NewDInt(tree.DInt(status.Flushes.Last.KVs)),
+				tree.NewDInt(tree.DInt(status.Flushes.Last.Bytes)),
+				dur(status.Flushes.Last.SlowestBatchNanos),
+				nullCur(age(time.UnixMicro(status.Flushes.Current.StartedUnixMicros))),
+				nullCur(tree.NewDInt(tree.DInt(status.Flushes.Current.TotalKVs))),
+				nullCur(tree.NewDInt(tree.DInt(status.Flushes.Current.ProcessedKVs))),
+				nullCur(tree.NewDInt(tree.DInt(status.Flushes.Current.Batches))),
+				nullCur(dur(status.Flushes.Last.SlowestBatchNanos)),
```
Comment (msbutler) [inline pkg/sql/crdb_internal.go:9272]: i think you meant to use `Current` here, not `Last`

**Review by msbutler — APPROVED** [review]

**Pull request merged**